src/Util.cpp

```diff
@@ -64,6 +64,66 @@ bool FileExists(const char *filename)
 #endif
 }
 
+// Get user-specific config dir manually.
+// apple:   ~/Library/Application Support/
+// windows: %APPDATA%
+// unix:    ${XDG_CONFIG_HOME:-~/.config}
+std::string get_xdg_user_config_home()
+{
+    std::string path;
+#ifdef __APPLE__
+    std::string home(getenv("HOME"));
+    path = home + "/Library/Application Support/";
+#elseif _WIN32
+    std::string app_data(getenv("APPDATA"));
```

Comment:
This should `getenv("LOCALAPPDATA")` to use `Local\` instead of `Roaming\`, we use `Local\` for the GUI as well.